docs/source/contents/4_draft_mapping.rst

```diff
@@ -1794,16 +1794,16 @@ genre
 | edm:hasType     | URI/String Literal | N/A   |
 +-----------------+--------------------+-------+
 
-genre[not(@*)]
---------------
+genre[not(@*)] AND genre[@authority='dct']
```

Comment:
Perhaps slightly rename this (since we aren't including ALL genre[not(@*)] values here)? Maybe something less Xpath focused like "Values that map to dcterms:type values"?

Reply:
Thank you!